### PR TITLE
Add UIE english model

### PR DESCRIPTION
**Pull request by linjieccc**
### PR types

New features
### PR changes

Models
### Description

Add UIE english model

### CODE REVIEW

File: docs/model_zoo/taskflow.md [hunk 1]
```diff
@@ -775,8 +772,9 @@ from paddlenlp import Taskflow
   | `uie-mini`| 6-layers, 384-hidden, 12-heads |
   | `uie-micro`| 4-layers, 384-hidden, 12-heads |
   | `uie-nano`| 4-layers, 312-hidden, 12-heads |
+  | `uie-base-en` | 12-layers, 768-hidden, 12-heads |
```
Comment (wawltor) [inline docs/model_zoo/taskflow.md]: 放在uie-base后面，同时加一列，备注一下是英文UIE模型

Comment (linjieccc) [reply docs/model_zoo/taskflow.md]: done, thx~

**Review by wawltor — DISMISSED** [review]
LGTM

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**